src/kcmd/grub.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/canonical/rt-conf/src/model"
 )
 
+const GrubDefaultFile = "/etc/default/grub"
```

Comment:
Why is the default grub file path still needed? Please do a thorough check and remove all code obsoleted by changes made in this PR.